src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const COST = 40;
+  const LONG_RENT = 7;
+  const SHORT_RENT = 3;
+  let discount = 0;
+
+  if (days >= LONG_RENT) {
+    discount = 50;
+  } else if (days >= SHORT_RENT) {
```

Comment:
4. [checklist] Prefer `if` with `return` over `if else` to simplify later conditions.

Reply:
> 4. [checklist] Prefer `if` with `return` over `if else` to simplify later conditions.

Good point